### PR TITLE
Fix colors in route viewer

### PR DESCRIPTION
**Pull request by amy-corson-ibigroup**
**Description:**

Fixing regression that is causing all route colors in route viewer to be set to #333333

Also took off a couple of pixels that were causing a weird crop on some routes with long names.

**PR Checklist:**
- [ ] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [ ] Are all languages supported (Internationalization/Localization)?
- [ ] Are appropriate Typescript types implemented?

| Before | After |
|--------|-------|
|image|image|

### CODE REVIEW

File: lib/components/narrative/metro/default-route-renderer.tsx [hunk 1]
```diff
@@ -37,7 +36,7 @@ const DefaultRouteRenderer = ({
   const routeTitle = leg.routeShortName || leg.routeLongName
   return (
     <Block
-      color={leg.routeColor || '333333'}
+      color={leg.origColor || '333333'}
```
Comment (miles-grant-ibigroup) [inline lib/components/narrative/metro/default-route-renderer.tsx:39]: Woah there! We need to include `routeColor` as a fallback!

File: lib/components/narrative/metro/default-route-renderer.tsx [hunk 2]
```diff
@@ -26,7 +25,7 @@ const Block = styled.span<{ color: string; isOnColoredBackground?: boolean }>`
 `
 
 export type RouteRendererProps = {
-  leg: Leg & { onColoredBackground?: boolean }
+  leg: Leg & { onColoredBackground?: boolean; origColor: string }
```
Comment (miles-grant-ibigroup) [inline lib/components/narrative/metro/default-route-renderer.tsx:28]:
```suggestion
  leg: Leg & { onColoredBackground?: boolean; origColor?: string }
```

`origColor` is optional!

**Review by miles-grant-ibigroup — CHANGES_REQUESTED** [review]
I have a few questions about this PR. Doesn't this defeat the point of route color overrides entirely? Since we're always using `origColor`

**Comment by amy-corson-ibigroup** [comment]
> I have a few questions about this PR. Doesn't this defeat the point of route color overrides entirely? Since we're always using `origColor`

Definitely open to other suggestions! Using the color variables that are available on `leg` this seemed like the easiest way to make sure the appropriate colors were showing up on each route.

**Comment by miles-grant-ibigroup** [comment]
> > I have a few questions about this PR. Doesn't this defeat the point of route color overrides entirely? Since we're always using `origColor`
> 
> Definitely open to other suggestions! Using the color variables that are available on `leg` this seemed like the easiest way to make sure the appropriate colors were showing up on each route.

Yes but doesn't this PR disable the route color override functionality? Since we're always using the `origColor`

**Comment by amy-corson-ibigroup** [comment]
I see what you mean, I think I fixed it!

**Review by miles-grant-ibigroup — CHANGES_REQUESTED** [review]
This still breaks the functionality! If we set a route color in the config, then don't we want it applied in the route viewer?

**Comment by amy-corson-ibigroup** [comment]
Okay this should fix the route viewer and allow color overrides by mode

**Review by miles-grant-ibigroup — APPROVED** [review]
Works perfectly in every situation I could throw at it. Well done! Thanks for dealing with all this back and forth. Should be good to go once the snapshots are updated!

**Review by binh-dam-ibigroup — APPROVED** [review]
LGTM

**Pull request merged**